sql/moz-fx-data-shared-prod/contextual_services_derived/event_aggregates_v1/init.sql

```diff
@@ -0,0 +1,21 @@
+CREATE OR REPLACE TABLE
+  contextual_services_derived.event_aggregates_v1
+PARTITION BY
+  submission_date
+CLUSTER BY
+  source,
+  event_type
```

Comment:
Note: Now that we support clustering definitions under the `bigquery` key in `metadata.yaml` and `schema.yaml` files, that approach is preferred over defining an `init.sql`. See https://mozilla.github.io/bigquery-etl/cookbooks/common_workflows/#adding-a-new-scheduled-query

Reply:
Ok thanks I remembered there was something like that but I couldn't find the documentation so I thought I got it mixed up with something else.